lib/algorand.transaction.crafter.spec.ts

```diff
@@ -22,21 +22,12 @@ export function concatArrays(...arrs: ArrayLike<number>[]) {
 }
 
 describe("Algorand Transaction Crafter", () => {
-	let algorandCrafter: AlgorandTransactionCrafter
-	let algoEncoder: AlgorandEncoder
-
 	const genesisId: string = "GENESIS_ID"
 	// genesis in base64
 	const genesisHash: string = Buffer.from(randomBytes(32)).toString("base64")
 
-	beforeEach(async () => {
-		algorandCrafter = new AlgorandTransactionCrafter(genesisId, genesisHash)
-		algoEncoder = new AlgorandEncoder()
-	})
-
-	afterEach(() => {
-		jest.resetAllMocks()
-	})
+	const algorandCrafter: AlgorandTransactionCrafter = new AlgorandTransactionCrafter(genesisId, genesisHash)
+	const algoEncoder: AlgorandEncoder = new AlgorandEncoder()
```

Comment:
We should keep the beforeEach / afterEach and construct each instance before / after each test. This because if the implementation ever changes in those objects, it might give false positives or break tests that should pass.